event-service-api/src/main/java/com/restapi/api/EventService.java

```diff
@@ -1,12 +1,49 @@
 package com.restapi.api;
 
+import com.restapi.dto.Event;
+
+import java.util.List;
+
 public interface EventService {
 
-    void createEvent();
-    void updateEvent();
-    void getEvent();
-    void deleteEvent();
-    void getAllEvents();
-    void getAllEventsByTitle();
+    /**
+     * Creates new event
+     * @param event
+     * @return
```

Comment:
if you add javadoc, fill out all the fields, here `@return` is not described
however, I would not in general add a javadoc to interfaces like this unless it is going to be used separately from the sources